backend/src/controllers/MyUserController.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import User from '../models/user';
+
+const createCurrentUser = async (req: Request, res: Response) => {
+  try {
+    const { auth0Id } = req.body;
+    const existingUser = await User.findOne({ auth0Id });
+    if (existingUser) {
+      return res.status(400).send('User already exists');
+    }
+    const newUser = new User(req.body);
+    await newUser.save();
+    res.status(201).send(newUser.toObject());
+  } catch (error) {
+    console.log(error);
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/14#issuecomment-2363140612>#d9c5f2</a><span> - 09/20/2024, 08:21 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Potential logging of sensitive information in the catch block
</div>
<div id="fix">
<b>Fix:</b> Replace the console.log(error) with a more secure logging approach. Use a logging library that allows for redaction of sensitive information and provides different log levels.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,5 +1,6 @@
  import { Request, Response } from 'express';
  import User from '../models/user';
 +import logger from '../utils/logger';
 
  const createCurrentUser = async (req: Request, res: Response) => {
    try {
 @@ -12,7 +13,7 @@
      await newUser.save();
      res.status(201).send(newUser.toObject());
    } catch (error) {
 -    console.log(error);
 +    logger.error('Error creating user', { error: error.message });
      res.status(500).send('Error creating user');
    }
  };
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged